gradle.properties

```diff
@@ -24,4 +24,4 @@
 #   permissions and limitations under the License.
 #
 
-version = 1.1.0
+version = 1.2.0
```

Comment:
I think this should be removed altogether, it's re-defined in https://github.com/opensearch-project/job-scheduler/pull/68/files#diff-49a96e7eea8a94af862798a45174e6ac43eb4f8b4bd40759b5da63ba31ec3ef7R88